### DISCUSSION TITLE
it is possible to use strapi in a static server(shared hosting) like godaddy

### DISCUSSION THREAD

**Discussion by JorgeMantillaHernandez** [strapi/strapi#6788] [Legacy Topics]
Hello, I am interested in creating a personal cms, but currently I have a static hosting with godaddy for two years, it is impossible to make a change at this time. I would like to know if I can deploy to use it on this server? Thank you

**Comment by robertpiosik** [comment]
Strapi requires Node.js runtime. Ask your hosting provider if you are able to use it within the product you pay for. Unfortunately in most cases term `hosting` is associated with php scripting environment.
